src/vr.zig

```diff
@@ -2364,7 +2380,7 @@ pub const Replica = struct {
             const checksum = self.journal.entry_for_op_exact(op).?.checksum;
 
             if (self.create_prepare_message(op, checksum)) |prepare| {
-                defer self.message_bus.unref(prepare);
+                defer self.message_bus.pool.unref(prepare);
```

Comment:
When we init MessageBus, this in turn hides away the init of MessagePool, so that `Replica.init()` only needs to know about the MessageBus and nothing within it.

Our new `message.ref()` style takes this even further so that call sites only need to know about a message.

However, when we need to get messages or `unref` from within `src/vr.zig`, we need to go through `message_bus.pool`.

What are your thoughts on keeping `MessageBus.unref()`, and letting this proxy instead to `MessagePool.unref()` so that MessagePool is entirely encapsulated from `src/vr.zig`? Same for the `get_message` etc. methods? There are not many to proxy so this is probably a good candidate for applying Demeter.

Reply:
Sounds fine to me, as you say it's only three methods and it would clean up the diff and make things less verbose.